src/mongo/db/pipeline/document_source_group.h

```diff
@@ -147,15 +147,21 @@ class DocumentSourceGroup final : public DocumentSource, public SplittableDocume
      */
     Value expandId(const Value& val);
 
+    struct Field {
+        std::string fieldName;
+        Accumulator::Factory accumulatorFactory;
+        boost::intrusive_ptr<Expression> expression;
+    };
+
     /**
-     * 'vFieldName' contains the field names for the result documents, 'vpAccumulatorFactory'
-     * contains the accumulator factories for the result documents, and 'vpExpression' contains the
-     * common expressions used by each instance of each accumulator in order to find the right-hand
-     * side of what gets added to the accumulator. These three vectors parallel each other.
+     * '_fieldName' contains the field names for the result documents, '_accumulatorFactory'
```

Comment:
Let's split this comment into three, one about each data member, and put those three comments next to their corresponding fields above. Also, we now tend to prefer the `// Comment` syntax for comments describing member variables (I think), so if you could update the `/** comment */` to be `// comments`, that would be great.

Edit: we already have a class for this! Please move these comments to https://github.com/mongodb/mongo/blob/r3.5.3/src/mongo/db/pipeline/accumulation_statement.h, inline with 'fieldName', 'factor', and 'expression' there.